### DISCUSSION TITLE
Can I just use only TCP protocol ？

### DISCUSSION THREAD

**Discussion by ttlib** [rustdesk/rustdesk#2984] [Q&A]
In an internal network, UDP protocol is forbidden. Can I use only TCP protocol?

**Comment by rustdesk** [comment]
Sadly, no for the time being.

**Comment by rustdesk** [comment]
but direct ip access uses TCP only.

**Reply by Chilkings** [reply]
same problem, how to config use tcp only?

**Reply by rustdesk** [reply]
Pro
